xds/third_party/envoy/import.sh

```diff
@@ -68,6 +70,8 @@ do
   cp -p "${tmpdir}/${SOURCE_PROTO_BASE_DIR}/${file}" "${file}"
 done
 
+# DO NOT TOUCH! The following section is upstreamed with an internal script.
+
```

Comment:
Indicate where the section ends. Otherwise LGTM.

Reply:
The section is everything below.